app/email_preference/migrations/0001_initial.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.2.24 on 2022-08-12 11:27
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+import economy.models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='EmailPreferenceLog',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_on', models.DateTimeField(db_index=True, default=economy.models.get_time)),
+                ('modified_on', models.DateTimeField(default=economy.models.get_time)),
+                ('user_id', models.CharField(blank=True, max_length=25, null=True)),
+                ('destination', models.CharField(default='hubspot', max_length=255)),
+                ('event_data', django.contrib.postgres.fields.jsonb.JSONField(blank=True, default=dict, null=True)),
+                ('response_data', django.contrib.postgres.fields.jsonb.JSONField(blank=True, default=dict, null=True)),
+                ('processed_at', models.DateTimeField(blank=True, null=True)),
```

Comment:
I note the other two fields are in the format _on, as apposed to _at.

Could we change this to "processed_on" for consistency.